python/cuml/test/test_kmeans.py

```diff
@@ -109,6 +109,45 @@ def test_kmeans_sequential_plus_plus_init(nrows, ncols, nclusters,
     assert abs(cu_score - sk_score) <= cluster_std * 1.5
 
 
+@pytest.mark.parametrize('nrows', [1000, 10000])
+@pytest.mark.parametrize('ncols', [10, 50])
+@pytest.mark.parametrize('nclusters', [5, 10])
+@pytest.mark.parametrize('max_weight', [10, 20])
+@pytest.mark.parametrize('random_state', [i for i in range(5)])
```

Comment:
How long does this test take to run? It looks like it has a large number of variations (80?) that we may want to cut down for efficiency .

Reply:
I have reduced the number of variations to 20 and kept it similar to other tests now.